src/SFA.DAS.EmployerAccounts.Jobs/Data/PopulateRepository.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Threading.Tasks;
+using Dapper;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.EmployerAccounts.Data;
+
+namespace SFA.DAS.EmployerAccounts.Jobs.Data
+{
+    public class PopulateRepository : IPopulateRepository
+    {
+        private readonly ILogger _logger;
+        private readonly EmployerAccountsDbContext _db;
+
+        public PopulateRepository(Lazy<EmployerAccountsDbContext> db, ILogger logger)
+        {
+            _logger = logger;
+            _db = db.Value;
```

Comment:
We could get more value out of the `Lazy<>` if we only referenced the `Value` just in time, rather than anticipatorily in the constructor.